markdown/ch5.markdown

```diff
@@ -577,14 +572,12 @@ were head of the NumPy Agency for Space Affairs and had to rotate
 millions of images streaming in from the newly launched Jupyter
 Orbiter?
 
-Of course, one option would be to rewrite all your code in C++, but
-let's presume you have diligently read Scott Meyers' excellent books
-and are deeply convinced that you will never write bug free C++ under
-pressure.  While options like Cython, Numba, or Julia are available,
-we'd like to show you a quick workaround using SciPy's sparse
-matrices.
+In such cases, you want to squeeze every ounce of performance from your
```

Comment:
I don't recall the TR comment, but why is this paragraph better than the previous version?